microshift_install_get_ready/microshift-install-get-ready.adoc

```diff
@@ -16,3 +16,70 @@ include::modules/microshift-install-system-requirements.adoc[leveloffset=+1]
+
+For every installation type, you must configure both {microshift-short} itself and the operating system. For example, determine the end-point hardware you want to run {op-system-bundle} on, your application storage needs, networking for cluster or application access, and your specific authentication and security needs. Where you want to run your cluster and what your application needs to do determines the {op-system-base} installation type that you choose.
+
+{microshift-short} is generally configured by adding parameters and values to the {microshift-short} YAML configuration file. See xref:../microshift_configuring/microshift-using-config-yaml.adoc#microshift-configuring[Using the {microshift-short} configuration file]. A {op-system-base} installation requires configuration specific to the type and tasks. Consult
+
+For most installation types, you must take the following steps:
+//Q: Do we use for all installation types? wasn't sure, so wrote "most", but the idea here is "all"
+
+* Be able to download the link:https://console.redhat.com/openshift/install/pull-secret[pull secret] from the Red Hat Hybrid Cloud Console.
+
+* Be ready to configure {microshift-short} by adding parameters and values to the {microshift-short} YAML configuration file. See xref:../microshift_configuring/microshift-using-config-yaml.adoc#microshift-configuring[Using the {microshift-short} configuration file]
+
+* Decide whether you need to configure storage for the application and tasks you are using in your {microshift-short} cluster, or decide to disable the {microshift-short} storage plug-in completely.
+** See link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/configuring_and_managing_logical_volumes/overview-of-logical-volume-management_configuring-and-managing-logical-volumes[Overview of logical volume management] for more information about creating volume groups and persistent volumes on {op-system-base}.
+** See xref:../microshift_storage/microshift-storage-plugin-overview.adoc#microshift-storage-plugin-overview[Dynamic storage using the LVMS plugin] for more information about the {microshift-short} plug-in.
+
+* Configure networking settings according to the access needs you plan for your {microshift-short} cluster and applications. Consider whether you want to use single or dual-stack networks, configure a firewall, or configure routes.
+** See xref:../microshift_networking/microshift-networking-settings.adoc#microshift-networking[Understanding networking settings] for more information about {microshift-short} networking options.
+//Q: should we have a RHEL link here as well?
+
+* Install the {oc-first} to access your cluster, see xref:../microshift_cli_ref/microshift-oc-cli-install.adoc#microshift-oc-cli-install[Installing the OpenShift CLI tool].
+
+* Understand the differences between the valid {op-system-base} installation types available, along with the support scope for each and the tools used.
```

Comment:
can we pull this block up and maybe even make it its own section heading, e.g. "Overview of available Installation types"?